src/CHANGES.txt

```diff
@@ -62,6 +62,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       prone to races. Add a hack to the PY2 version (from Issue #3351) to
       be less prone to a race in the check for old-style cache.
     - Fix coding error in docbook tool only exercised when using python lxml 
+    - Recognize two additional GNU compiler header directory options:
```

Comment:
Add two .. to ParseFlags()... so people know to what this applies?  Also do the docs cover any of the supported flags, if so probably should add these two to the docs?

Reply:
Well, the other option that was added this way, --isystem`, doesn't seem to have a mention in docs. Doesn't mean it should not. I'll take a look eventually.

Reply:
There was a place in the manpage that needed updating, yes (it was never updated with `-isystem` either is why I didn't find it at first).